ui/components/app/edit-gas-fee-popover/edit-gas-item/edit-gas-item.js

```diff
@@ -90,6 +101,41 @@ const EditGasItem = ({ priorityLevel, onClose }) => {
     onClose();
   };
 
```

Comment:
@adonesky1 : I would prefer to have here s separate small functional component that returns tooltip.